include/boost/geometry/algorithms/detail/overlay/graph/is_target_operation.hpp

```diff
@@ -154,18 +159,23 @@ bool is_better_collinear_for_intersection(Operation const& op, Operation const&
     }
 
     return op.enriched.ahead_distance_of_side_change
-        <= other_op.enriched.ahead_distance_of_side_change;
+           <= other_op.enriched.ahead_distance_of_side_change;
 }
 
 template <operation_type Operation>
-struct is_better_collinear_target {};
+struct is_better_collinear_target
+{
+};
```

Comment:
This is somehow split (though I have `AllowShortBlocksOnASingleLine`)

Reply:
With the more detailled Brace rules as in https://github.com/boostorg/geometry/pull/1400#issuecomment-2833213438 , I can get 

```
template <operation_type Operation>
struct is_better_collinear_target
{};
```